scripts/init_mkcert.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [[ ! -d ./docker-nginx/certs ]]; then
```

Comment:
```suggestion
if [[ ! -d ./docker-nginx ]]; then
```
`./docker-nginx/certs` does not exist in a qfieldcloud installation from scratch, checking for `./docker-nginx` should be sufficient.